doc/admin/pool_operations.md

```diff
@@ -12,35 +12,30 @@ manage storage pools from the command line.
 
 **To create a pool:**
 ```bash
-$ dmg pool create --size=NTB
+$ dmg pool create --total-size=NTB
```

Comment:
`total-size` is redundant. Please just use `size`, as this is the parameter name that was chosen for the simplified pool create workflow.

Reply:
Okay.